internal/connection_manager.go

```diff
@@ -46,13 +46,13 @@ func newConnectionManager(client *HazelcastClient) *connectionManager {
 	return &cm
 }
 
-func (cm *connectionManager) nextConnectionID() int64 {
+func (cm *connectionManager) NextConnectionID() int64 {
```

Comment:
As far as I see this is made public just for the sake of tests. I cant think of any better way off the top of my head other than putting them into the same package, which is complicated for now.